### PR TITLE
[ADD] Create new board test automation

### PR DESCRIPTION
**Pull request by annaslodkowska**

### CODE REVIEW

File: src/test/java/com/patronage/backend/boards/BoardsTest.java [hunk 1]
```diff
@@ -2,13 +2,27 @@
...
 import org.junit.Test;
 
 import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
 
 public class BoardsTest extends BaseAuthTest {
 
+    private static final String VALID_BOARD_NAME = "board name";
+    private static final String VALID_BOARD_REQUEST_BODY = "{\"name\": \"" + VALID_BOARD_NAME + "\"}";
```
Comment (MarekPawlowskiQA) [inline src/test/java/com/patronage/backend/boards/BoardsTest.java:16]: string format will be more redable here i think

File: src/test/java/com/patronage/backend/boards/BoardsTest.java [hunk 2]
```diff
@@ -2,13 +2,27 @@
...
+
+    public RequestSpecification givenBodyJson(String json) {
+        return given().header("Content-Type", "application/json").body(json);
```
Comment (ospatrycja) [inline src/test/java/com/patronage/backend/boards/BoardsTest.java:19]: I think in our API we will always use content-type - application/json. Maybe we should move this function to BaseTest?

File: src/test/java/com/patronage/backend/boards/BoardsTest.java (hunk 1 above)
Comment (annaslodkowska) [reply src/test/java/com/patronage/backend/boards/BoardsTest.java]: I agree, fix done

File: src/test/java/com/patronage/backend/boards/BoardsTest.java (hunk 2 above)
Comment (annaslodkowska) [reply src/test/java/com/patronage/backend/boards/BoardsTest.java]: I agree, fix done

**Review by MarekPawlowskiQA — APPROVED** [review]
lgtm :)

**Review by ospatrycja — APPROVED** [review]
lgtm :)

**Pull request merged**